lib/active_shipping/carrier.rb

```diff
@@ -92,6 +92,16 @@ def find_tracking_info(tracking_number, options = {})
       raise NotImplementedError, "#find_tracking_info is not supported by #{self.class.name}."
     end
 
+    # Get a list of services available for the a specific route
+    #
+    # @param origin_country [String] The country of origin
```

Comment:
Agreed - I want to be clear with using `name` vs `code` suffixes. There's a time and place for a `name`, and for `code`, and in general I'd say the former is for presentation and the latter is for logic.

Reason being, If you follow `ISO 3166-1` standards (or something similar), you'll never run into instances where you have two representations of the same country (e.g. `North Korea` vs `Democratic People's Republic of Korea`, where the code is always `KP`). It doesn't matter whether you call it North Korea or DPRK in your logic, because your logic is only concerned with `KP`. It's only when you surface it that it "matters".

TLDR use `_code`, we'll remove any ambiguity in `Location.country` as well - a general theme in v2.